assets/scss/components/_waiting.scss

```diff
@@ -0,0 +1,19 @@
+/*
+Waiting State
+
+Markup:
+<span class="waiting"></span>
+
+Styleguide Waiting State
```

Comment:
As this is only inserted by JS I don't think we should have this documented in the component library. People might try and add it directly in the markup otherwise.